models/nexusmutual/ethereum/nexusmutual_ethereum_quotation_trades.sql

```diff
@@ -0,0 +1,99 @@
+{{ config(
+	alias ='trades',
+	partition_by = ['block_date'],
+	materialized = 'incremental',
+	file_format = 'delta',
+	incremental_strategy = 'merge',
+	unique_key = ['block_date', 'cover_block_number', 'status_num', 'evt_tx_hash', 'evt_index'],
+    post_hook='{{ expose_spells(\'["ethereum"]\',
+                                "project",
+                                "nexusmutual",
+                                \'["sharkxff"]\') }}'
+	)
+}}
+
+{% set project_start_date = '2019-07-12' %}
+
+WITH quo_evt AS (
+    SELECT cid,
+           contract_address,
+           evt_block_number,
+           evt_block_time,
+           evt_index,
+           evt_tx_hash,
+           curr,
+           expiry,
+           premium,
+           premiumNXM,
+           scAdd,
+           sumAssured,
+           '0xd7c49cee7e9188cca6ad8ff264c1da2e69d4cf3b' as token
+    FROM
+        {{ source('nexusmutual_ethereum', 'QuotationData_evt_CoverDetailsEvent') }}
+    {% if not is_incremental() %}
+    WHERE evt_block_time >= '{{project_start_date}}'
+    {% endif %}
+    {% if is_incremental() %}
+    WHERE evt_block_time >= date_trunc("day", now() - interval '1 week')
+    {% endif %}
+),
+quo_cse_data as (
+    select * FROM (
+        select  *,
+            row_number() over(partition by cid order by evt_block_time desc) as time_rank
+        FROM {{ source('nexusmutual_ethereum', 'QuotationData_evt_CoverStatusEvent') }}
+    ) as quo_cse
```

Comment:
Please try to avoid using `select *` for performance gain, unless all columns needed(In that case you still can list out all columns and make it clearer for review and maintainance).